gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/RandomVertexEdgeSampling.java

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling;
+
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.model.api.operators.UnaryGraphToGraphOperator;
+
+import java.security.InvalidParameterException;
+
+/**
+ * Computes an edge sampling of the graph. First selects randomly chosen vertices of a given
+ * relative amount and all edges which source- and target-vertices where chosen. Then randomly
+ * chooses edges from this set of edges and their associated source- and target-vertices.
+ * No unconnected vertices will retain in the sampled graph.
+ */
+public class RandomVertexEdgeSampling implements UnaryGraphToGraphOperator {
+
+  /**
+   * The samping type enum
+   */
+  public enum VertexEdgeSamplingType {
+    /**
+     * Simple version (uniform version for both vertices and edges
+     */
+    SimpleVersion,
+    /**
+     * Nonuniform vertex sampling and then uniform edge sampling
+     */
+    NonuniformVersion,
+    /**
+     * Nonuniform vertex sampling and sample sizes add up to 1
+     */
+    NonuniformHybridVersion,
+  }
+
+  /**
+   * The sampling type
+   */
+  private final VertexEdgeSamplingType vertexEdgeSamplingType;
+
+  /**
+   * Relative amount of vertices in the result graph
+   */
+  private final float vertexSampleSize;
+
+  /**
+   * Relative amount of edges in the result graph
+   */
+  private final float edgeSampleSize;
+
+  /**
+   * Seed for the random number generator
+   * If seed is 0, the random generator is created without seed
+   */
+  private final long randomSeed;
+
+  /**
+   * Creates new RandomVertexEdgeSampling instance.
+   *
+   * @param sampleSize relative sample size for edges and vertices
```

Comment:
Please add an example value to the description: e.g `@param sampleSize relative sample size for edges and vertices, e.g. 0.5` that users know which value will be expected. Same for other params.